WordPress/Classes/ViewRelated/Reader/Controllers/ReaderDiscoverViewController.swift

```diff
@@ -42,8 +44,29 @@ class ReaderDiscoverViewController: UIViewController, ReaderDiscoverHeaderViewDe
         showSelectInterestsIfNeeded()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        setupNotificationsBarButtonItem()
+    }
+
     private func setupNavigation() {
         navigationItem.largeTitleDisplayMode = .never
+        setupNotificationsBarButtonItem()
+    }
+
+    private func setupNotificationsBarButtonItem() {
+        notificationsButtonCancellable = nil
+        if traitCollection.horizontalSizeClass == .regular {
+            notificationsButtonCancellable =         notificationsButtonViewModel.$image.sink { [weak self] in
```

Comment:
There are a few whitespaces after `=`.

Reply:
Thanks; didn't notice it with the way my editor is set up. Fixed.